### PR TITLE
Update parachain registration script to new extrinsic

### PR DESCRIPTION
**Pull request by rphmeier**
Tested locally.

Usage: `{url} {wasm_file} {genesis_head} {para_id} {types_file}`

In practice: `scripts/register_parachain.sh ws://127.0.0.1:9944 "//Alice" ~/adder_genesis.wasm $(cat ~/adder_genesis.head) 100 scripts/temp_parachain_types.json`

I removed the balance setting thing as that's a separate concern and it didn't work for some reason.

### CODE REVIEW

File: scripts/temp_parachain_types.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (bkchr) [inline scripts/temp_parachain_types.json:1]: @jacogr can you upstream these types please.

**Review by bkchr — APPROVED** [review]

File: scripts/temp_parachain_types.json (hunk 1 above)
Comment (jacogr) [reply scripts/temp_parachain_types.json:1]: https://github.com/polkadot-js/api/pull/2895

Comment (jacogr) [reply scripts/temp_parachain_types.json:1]: And the apps UI API bump - https://github.com/polkadot-js/apps/pull/4136

File: scripts/register_parachain.sh [hunk 2]
```diff
@@ -11,46 +11,40 @@ seed=$2
...
     --sudo \
     --seed "${seed?}" \
-    tx.balances.setBalance \
-        "${account?}" \
-        $((tokens * 10 ** 12)) \
-        0
+    --types "${types?}" \
+    tx.parasSudoWrapper.sudoScheduleParaInitialize \
+        "${parachain_id?}" \
+        "{ \"genesisHead\":\"${genesis?}\", \"validationCode\":\"${wasm?}\", \"parachain\": true }" \
```
Comment (JoshOrndorff) [inline scripts/register_parachain.sh:50]: Doesn't this put the entire ~1MB wasm blob on the CLI? Was your shell able to handle that?

Comment (rphmeier) [reply scripts/register_parachain.sh:50]: Shell should be able to handle 1MB easily. Many terminal emulators might not. Just send output of the script to `/dev/null` if it's a problem.

Comment (menduong) [reply scripts/register_parachain.sh:50]: I try 
./register.sh ws://127.0.0.1:9944 "//Alice" ~/genesis.wasm $(cat ~/genesis-state) 14 temp_parachain_types.json > /dev/null 2>&1
but nothing happened in my network

**Pull request merged**